stratum/hal/lib/common/p4_service.cc

```diff
@@ -547,21 +542,23 @@ ::grpc::Status P4Service::StreamChannel(
   // 3- At any point of time, only the master stream is capable of sending
   //    and receiving packets.
 
-  // First thing to do is to find a new ID for this connection.
-  auto ret = FindNewConnectionId();
+  // First thing to do is to check the connection limit and increment it.
```

Comment:
it is not obvious to me what connection limit means. Maybe add an additional comment saying that we don't handle more than N connections at time